docs/src/docs/asciidoc/user/caching-concepts.adoc

```diff
@@ -40,7 +40,7 @@ and technical decision based upon the requirements and assumptions of your appli
 
 
 === Storage Tiers
-You can configure Ehcache to use various data storage areas.  When a cache is configured to use more than one data
+[[storage-tiers]]You can configure Ehcache to use various data storage areas.  When a cache is configured to use more than one data
```

Comment:
Indeed a good practice to explicitly create them - but needs to be on top of the section definition, not below it.